README.rst

```diff
@@ -0,0 +1,110 @@
+clang-tools Introduction
+========================
+
+.. image:: https://img.shields.io/pypi/v/clang-tools
+    :target: https://pypi.org/project/clang-tools/
+    :alt: PyPI
+.. image:: https://github.com/cpp-linter/clang-tools-pip/actions/workflows/python-test.yml/badge.svg
+    :target: https://github.com/cpp-linter/clang-tools-pip/actions/workflows/python-test.yml
+    :alt: Python test
+.. image:: https://codecov.io/gh/cpp-linter/clang-tools-pip/branch/main/graph/badge.svg?token=40G5ZOIRRR
+    :target: https://codecov.io/gh/cpp-linter/clang-tools-pip
+    :alt: codecov
+.. image:: https://img.shields.io/badge/platform-linux--64%20%7C%20win--64%20%7C%20osx--64%20-blue
+    :target: https://pypi.org/project/clang-tools/)
+    :alt: Platfrom
+.. image:: https://img.shields.io/pypi/dw/clang-tools
+    :alt: PyPI - Downloads
+
+
+Install clang-tools binaries (clang-format, clang-tidy) with pip.
+
+Install
+-------
+
+Install `clang-tools` command with pip
+
+.. code-block:: shell
+
+    pip install clang-tools
+
+Install `clang-tools` from git repo
+
+.. code-block:: shell
+
+    pip install git+https://github.com/cpp-linter/clang-tools-pip.git@main
+
+Usage
+-----
+
+.. code-block:: shell
+
+    clang-tools --help
```

Comment:
The new usage should be 

```bash
usage: clang-tools [-h] [-i INSTALL] [-d DIRECTORY] [-f]

optional arguments:
  -h, --help            show this help message and exit
  -i INSTALL, --install INSTALL
                        Install clang-tools with specific version. default is 13.
  -d DIRECTORY, --directory DIRECTORY
                        The directory where is the clang-tools install.
  -f                    Force overwriting the symlink to the installed binary. This will only overwrite an existing
                        symlink.
```

Reply:
good catch! I updated it.